scripts/coverage-check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+THRESHOLD='85.0'
```

Comment:
is this copy pasted? if it's actually higher, we can change it here

Reply:
yes, it is now 100%, maybe I can set it to `90`?